porcupine/_state.py

```diff
@@ -7,9 +7,15 @@
 import tkinter
 import types
 from typing import Any, Type
+from sys import platform
 
 from porcupine import images, tabs, utils
 
+if platform == "win32":
+    from ctypes import windll
+
+    windll.shcore.SetProcessDpiAwareness(1)
```

Comment:
On Windows 7, I get this error:

![win7](https://user-images.githubusercontent.com/18505570/168422046-aad99068-8fb1-49af-a385-6fbe66342c01.png)

I know that Windows 7 is no longer supported, but I use it whenever I develop Windows things for Porcupine, so I want it to still work.

```suggestion
    try:
        windll.shcore.SetProcessDpiAwareness(1)
    except FileNotFoundError:  # windows 7
        pass
```